src/flake8/options/aggregator.py

```diff
@@ -43,6 +45,16 @@ def aggregate_options(
             assert isinstance(dest_val, str)
             dest_name = dest_val
 
+        if config_name in ["ignore", "extend-ignore"]:
```

Comment:
should use a set containment rather than list containment here -- also I think this might be too late in the process -- would prefer this get validated during argument / config parsing if possible

Reply:
Also a good point, there is an earlier opportunity to check this immediately after option values have been normalized to a list of strings in config.py